Taarafo.Portal.Web/Taarafo.Portal.Web.csproj

```diff
@@ -1,22 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
-  <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+	</PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="PrettyBlazor" Version="1.2.0" />
-    <PackageReference Include="RESTFulSense" Version="2.4.0" />
-    <PackageReference Include="Syncfusion.Blazor.Themes" Version="19.4.0.38" />
-    <PackageReference Include="Syncfusion.Licensing" Version="19.4.0.38" />
-    <PackageReference Include="Xeption" Version="1.7.0" />
-    <PackageReference Include="Syncfusion.Blazor" Version="19.4.0.38" />
-  </ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="PrettyBlazor" Version="1.2.0" />
+		<PackageReference Include="RESTFulSense" Version="2.4.0" />
+		<PackageReference Include="Syncfusion.Blazor.Themes" Version="19.4.0.38" />
+		<PackageReference Include="Syncfusion.Licensing" Version="19.4.0.38" />
+		<PackageReference Include="Xeption" Version="1.7.0" />
+		<PackageReference Include="Syncfusion.Blazor" Version="19.4.0.38" />
+	</ItemGroup>
 
-  <ItemGroup>
-    <Content Update="wwwroot\imgs\error.png">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
+	<ItemGroup>
+		<Content Update="wwwroot\imgs\error.png">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+		</Content>
+	</ItemGroup>
```

Comment:
This indentation is not the default. Why is it too far out?

Reply:
@hassanhabib this is default for Tab indentstion.  It looks like the projects was initially setup with Indent Size = 4 and Insert Spaces rather than Tabs.

I have done a reset of my settings and upon format Ctrl+K, Ctrl+D it still formats the it with tabs.
![image](https://user-images.githubusercontent.com/797750/147737707-529a78f7-ee1e-48d2-93de-9a0c8310ae3a.png)

I also get the same outcome resetting the settings via UI
![image](https://user-images.githubusercontent.com/797750/147738271-543e14db-f2d6-4ad7-bcd0-9491c725e1d5.png)


Can you please double check behaviour on **Taarafo.Portal.Web.Tests.Unit.csproj** (or any other project) when you do a  Ctrl+K, Ctrl+D to format the doc?